api/setupTestDb.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+const { MongoMemoryServer } = require("mongodb-memory-server");
+
+let mongo = null;
+const connectionString = "mongodb://127.0.0.1:27017/test-db";
+
+const connectDB = async () => {
+  // some connection seems to be persisting,
+  // manually killing for now
+  // TODO: set it up so you can delete L11 and mongoose connects properly
```

Comment:
Could you elaborate on this issue you ran into a bit more? I'm curious :)

Reply:
(answered in person)